### DISCUSSION TITLE
How to key frame change in color of individual letters and maintain text effect?

### DISCUSSION THREAD

**Discussion by AlmostMikhail** [friction2d/friction#395] [Q&A] [answered]
I want to key frame the color of individual letters in a text but if I convert the text to a path won't that destroy the text effect. The text effect change the scale of the text over time. I want to have both color change of individual letters over time of a word and use the text effect.

**Comment by MaurycyLiebner** [comment, marked as answer]
You could keep the text effect and change colors using masks (separate objects) and blend modes.

**Reply by AlmostMikhail** [reply]
Thanks for the answer. Have you also moved to friction?

**Reply by MaurycyLiebner** [reply]
No. I silently follow Friction's development, but I have yet to try it.

**Reply by AlmostMikhail** [reply]
So, why did you stop development of enve? Such a great software. Without it friction would be nowhere. People saying enve is dead for years now.
